frontend/components/DuccStat.tsx

```diff
@@ -1,7 +1,20 @@
+import userHash from "../data/UserHash";
+
 export default function DuccStat(props: IDuccStat) {
+  {
+    console.log(props.user, props.user.length, props.user.trim().length);
```

Comment:
Remove this before the merge or is it more than just a debug log?

Reply:
removed, not necessary as I was only testing and forget to delete